Sources/LogConsole/LogInterceptor.cs

```diff
@@ -10,6 +10,8 @@
 using System;
 using UnityEngine;
 
+using Logger = KSPDev.LogUtils.Logger;
```

Comment:
I had a (very) quick look around for the Debug.LogXXXFormat() functions you mentioned but couldn't find them. Happy to help out a little although if you know what you want to do and how to do it it might be quicker and easier for you to just crack on with it when you get to it :)

My KSP modding experience is .. rudimentary.

However, the mod I'm minding at the moment also has its own logging subsystem so if there's a better way to do logging nowadays I'd like to learn about it.